usage/archetypes/quickstart/src/brooklyn-sample/README.md

```diff
@@ -65,9 +65,9 @@ deployment depending on what the application needs.
 
 For more information consider:
 
-* Visiting the open-source Brooklyn home page at  http://brooklyncentral.github.com
-* Forking the Brooklyn project at  http://github.com/brooklyncentral/brooklyn
-* Emailing  brooklyn-users@googlegroups.com 
+* Visiting the Apache Brooklyn home page at  http://brooklyn.io/
```

Comment:
I think we should use `htts://brooklyn.incubator.apache.org` (rather than this re-direct, even though it's shorter).